.gitlab-ci.yml

```diff
@@ -128,11 +128,11 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v4452649-b69b5d5
-  DATADOG_AGENT_BUILDERS: v4529511-a0d9303
-  DATADOG_AGENT_WINBUILDIMAGES: v4319944-68f7e12
-  DATADOG_AGENT_ARMBUILDIMAGES: v4319944-68f7e12
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v4319944-68f7e12
+  DATADOG_AGENT_BUILDIMAGES: v4877276-6d40aeb
+  DATADOG_AGENT_BUILDERS: v4877845-105f21c
+  DATADOG_AGENT_WINBUILDIMAGES: v4877276-6d40aeb
+  DATADOG_AGENT_ARMBUILDIMAGES: v4877276-6d40aeb
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v4877276-6d40aeb
```

Comment:
please update these to the image versions built from master once DataDog/datadog-agent-buildimages#144 and DataDog/datadog-agent-builders#203 are merged